Vagrantfile

```diff
@@ -57,7 +57,7 @@ def packages_darwin
     brew update > /dev/null
     brew install pkg-config readline openssl@1.1 zstd lz4 xz fakeroot git
     brew tap homebrew/cask
```

Comment:
is line 59 still needed? @samuel-w @m3nu

Reply:
Not needed any more. It's now integrated with the main Homebrew. From the [official GH page](https://github.com/Homebrew/homebrew-cask):

> To start using Homebrew Cask, you just need Homebrew installed.

Reply:
ok, thanks, i'll remove the line.